var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -54,17 +56,14 @@ class Kaldi(Package):    # Does not use Autotools
     depends_on('sctk', type='run')
     depends_on('speex', type='run')
     depends_on('openfst@1.4.1-patch', when='@c024e8')
-    depends_on('openfst')
```

Comment:
You should keep this line, otherwise new versions that get added won't depend on `openfst`, and you won't be able to select a dependency version on the command-line.